src/mono/wasm/runtime-test.js

```diff
@@ -261,6 +261,11 @@ var App = {
 			init ("");
 		}
 
+		if (args.length == 0) {
+			fail_exec ("Missing required --run argument");
```

Comment:
can't we just fall back to the entry point on the first assembly here if run is unset?

Reply:
I'm thinking of the src/mono/netcore/sample/wasm here and being able to serve that index.html directly

Reply:
This argument has the name of the main assembly, like in:
`dotnet wasm test-browser --app=. --output-directory=xharness-output -- --run WasmTestRunner.dll System.Buffers.Tests.dll`

By this point the html is already loaded, and this js is executing. Is there any other way to find a fallback assembly to use?

Reply:
`wasm test-browser` serves `index.html` by default. That can be changed on the command line with `wasm test-browser --html-file=foo.html`.

Reply:
I was thinking something like 

```
		if (args.length == 0) {
            let assemblies = config.assets.filter (function (a) { return a.behavior == "assembly"});
            args = ["--run", assemblies[assemblies.length - 1].name];
		}
```

Reply:
Maybe with some debug spew

Reply:
The better fix is probably making WasmAppBuilder set something like "entry-assembly" instead of "assembly" on the main assembly then allowing --run to override the entrypoint.

Reply:
I'll open a separate PR for this.